source/sayAllHandler.py

```diff
@@ -169,8 +169,7 @@ def _onLineReached(obj=self.reader.obj, state=state):
 			reason=controlTypes.REASON_SAYALL,
 			useCache=state
 		)
-		speechGen = speech.GeneratorWithReturn(speechGen)

```

Comment:
This wasn't needed, since the return is never considered. Instead we look at the return from `speakWithoutPauses` to see if it spoke.